`azurerm_web_application_firewall_policy`: ruleset support `Microsoft_DefaultRuleSet_2.1` (#23818)
hashicorp/terraform-provider-azurerm: website/docs/r/web_application_firewall_policy.html.markdown

The `excluded_rule_set` block supports the following:

> * `type` - (Optional) The rule set type. The only possible value is `Microsoft_DefaultRuleSet` and `OWASP`. Defaults to `OWASP`.

Reviewer: ```suggestion
* `type` - (Optional) The rule set type. Possible values include `Microsoft_DefaultRuleSet` and `OWASP`. Defaults to `OWASP`.
```

Author: updated.

> * `version` - (Optional) The rule set version. The only possible value is `2.1` (for rule set type `Microsoft_DefaultRuleSet`) and `3.2` (for rule set type `OWASP`). Defaults to `3.2`.

Author: updated.
